test/features/formItemExtra.test.ts

```diff
@@ -0,0 +1,31 @@
+import faker from 'faker';
+
+import { Tester } from '@mighty-justice/tester';
+
+import { COMPONENT_GENERATORS } from '../factories';
+
+const SUPPORTING_COMPONENTS = [
+  'Form',
+  'FormCard',
+  'FormDrawer',
+  'FormModal',
+];
+
+// Tests that users can pass in optional extraFieldConfig props to render an extra under form item.
+describe('formItemExtra', () => {
+  SUPPORTING_COMPONENTS.forEach(componentName => {
+    it(`Form item extras correctly renders in ${componentName}`, async () => {
+      const { ComponentClass, propsFactory } = COMPONENT_GENERATORS[componentName]
+        , name = faker.lorem.sentence()
+        , name2 = faker.lorem.sentence()
+        , props = propsFactory.build({
+            fieldSets: [[{ field: 'name'
+            , formItemProps: { extraFieldConfig: {label: 'ExtraField', showLabel: true, tooltip: null} } }]],
```

Comment:
Would you have to include `showLabel: true` and `tooltip: null` for this to work as intended? It seems like that should be the default